person.rb

```diff
@@ -1,10 +1,18 @@
 require_relative 'corrector'
+require './rental'
 
 class Person
   attr_accessor :name, :age
   attr_reader :id
```

Comment:
- [ ] There should be getter and setter (attr_accessor) for :rentals. Kindly add this.

Reply:
- [ ] The issue here has been properly resolved